Automattic/wp-calypso
# Seller Experience: Add Woo upsell to My Home banner

#### Changes proposed in this Pull Request

This adds a new post-launch card for upselling simple sellers on upgrading to Woo.

![image](https://user-images.githubusercontent.com/917632/156602480-7a6c8976-2a0b-4257-b27c-b9e596ba4706.png)

![image](https://user-images.githubusercontent.com/917632/156602496-22b378df-aa2c-42b5-b48b-8dd1d94ec2d8.png)

#### Testing instructions

- Switch to this PR
- Apply D76010-code to your sandbox and sandbox `public-api.wordpress.com`
- Visit "My Home" for one of your sites and append the ?view=VIEW_POST_LAUNCH query string parameter
- You should see a "Need more from your store?" card. (Note: You may need to click 'Next' to go through all of the other cards in the carousel before you see it.)


Related to #61471
